### DISCUSSION TITLE
How to write custom error codes in uws like 400, 401, 500

### DISCUSSION THREAD

**Discussion by perfogic** [uNetworking/uWebSockets.js#1052] [Q&A]
Hello, i have write error code like this but this still return 200 OK. Any help ?, thanks for your time!
<img width="440" alt="image" src="https://github.com/uNetworking/uWebSockets.js/assets/74671798/904385cb-d343-49ee-b304-1a5d67042788">
<img width="464" alt="image" src="https://github.com/uNetworking/uWebSockets.js/assets/74671798/172557b3-3074-434f-95b8-d13c60f40972">
<img width="664" alt="image" src="https://github.com/uNetworking/uWebSockets.js/assets/74671798/76e5b722-7bc7-4074-ac7f-af2b8c959a6e">

**Comment by uNetworkingAB** [comment]
Can you paste a minimal reproducer against curl

**Reply by perfogic** [reply]
hi, thanks for response. Can you tell me which is reproducer that you mention ^ ^. Do you mean the code or sth else

**Reply by uNetworkingAB** [reply]
Can you post a snippet of code that reproduces the problem. In text, not image and in full.

**Reply by perfogic** [reply]
Here mate: https://github.com/oraichain/oraidex-sdk/blob/feat/ibc-routing/packages/ibc-routing/src/apis/index.ts, https://github.com/oraichain/oraidex-sdk/blob/feat/ibc-routing/packages/ibc-routing/src/index.ts. Thanks for replying!
